packages/headless/src/features/analytics/analytics-utils.ts

```diff
@@ -775,11 +775,12 @@ export const validateProductRecommendationPayload = (
   productRec: ProductRecommendation
 ) => new Schema(productRecommendationPartialDefinition).validate(productRec);
 
-async function logNextEvent<PayloadType extends RelayPayload>(
+async function logNextEvent<PayloadType>(
   emitEvent: ReturnType<typeof createRelay>['emit'],
   type: string,
   payload: PayloadType
 ): Promise<void> {
+  //@ts-ignore
```

Comment:
This ts-ignore will be removed once Relay adjust the emitEvent function. It currently uses the RelayPayload type internally

Reply:
Hi Alex, I've created [this jira](https://coveord.atlassian.net/browse/LENS-1349). Could you add more details about the error on the jira? Could it be because the interface is using `unknown` rather than `any`?

Reply:
Good work've been done on Relay side.
Morphed the ignore into an except-error, the flag will get rise when no error occurs like this.
We can revisit then when the flag is raised/relay is updated. Merging.